src/sidebar-container.component.ts

```diff
@@ -57,9 +59,10 @@ import { Sidebar } from './sidebar.component';
 })
 export class SidebarContainer implements AfterContentInit, OnDestroy {
   @Input() backdropClass: string;
+  @Input() allowSidebarBackdropControl: boolean = true;
 
-  /** @internal */
-  _showBackdrop: boolean = false;
+  @Input() showBackdrop: boolean = false;
+  @Output() showBackdropChange = new EventEmitter<null>();
```

Comment:
I'm fairly confident that this should be `new EventEmitter<boolean>()`, and you need to actually emit it manually within `ngOnChange`.